SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/DummyObject.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#include "Interop/SpatialWorkerFlags.h"
+
+#include "DummyObject.generated.h"
+
+
+UCLASS()
```

Comment:
It looks more like a Spy object to me, since it records some information on calls to its functions, that is later used in checks.